gazebo_msgs/package.xml

```diff
@@ -1,36 +1,35 @@
 <?xml version="1.0"?>
-<package format="2">
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
   <name>gazebo_msgs</name>
   <version>2.8.4</version>
   <description>
-    Message and service data structures for interacting with Gazebo from ROS.
+    Message and service data structures for interacting with Gazebo from ROS2.
   </description>
 
   <maintainer email="jrivero@osrfoundation.org">Jose Luis Rivero</maintainer>
 
   <license>BSD</license>
 
-  <url type="website">http://gazebosim.org/tutorials?cat=connect_ros</url>
-  <url type="bugtracker">https://github.com/ros-simulation/gazebo_ros_pkgs/issues</url>
-  <url type="repository">https://github.com/ros-simulation/gazebo_ros_pkgs</url>
-
-  <author>John Hsu</author>
-
-  <buildtool_depend>catkin</buildtool_depend>
+  <buildtool_depend>ament_cmake</buildtool_depend>
 
+  <build_depend>rosidl_default_generators</build_depend>
```

Comment:
Should this be `buildtool_depend`?

Reply:
Switched, that seems like the right way to do it